rtree/index.py

```diff
@@ -23,12 +23,11 @@
 
 __c_api_version__ = core.rt.SIDX_Version()
 
-major_version, minor_version, patch_version = [
```

Comment:
Tests are failing because I removed these attributes. Is this something you want to keep?